praw/models/reddit/subreddit.py

```diff
@@ -1322,6 +1328,11 @@ def update(
             ``'light'`` or ``'dark'``.
         :param mod_only: (boolean) Indicate if the flair can only be used by
             moderators.
+        :param allowable_content: If specified, most be one of ``'all'``,
+            ``'emoji'``, or ``'text'`` to restrict content to that type.
+            If set to ``'emoji'`` then the ``'text'`` param must be a
+            valid emoji string, for example ``':snoo:'``.
+        :param max_emojis: (int) Maximum emojis in the flair (default: 10).
```

Comment:
Can you drop the default from here since that should represent what we set the default to? Perhaps you can instead say something like `Reddit defaults this value to 10`.